tests/component/test_export_signal_properties.py

```diff
@@ -2,62 +2,81 @@
 
 import pytest
 
+import nidaqmx
 from nidaqmx.constants import ExportAction
 from nidaqmx.error_codes import DAQmxErrors
 from nidaqmx.errors import DaqError
 from nidaqmx.task import Task
 
 
-def test__export_signal__get_enum_property__returns_value(ai_voltage_chan_task: Task):
+@pytest.fixture(scope="module")
+def ai_voltage_task(any_x_series_device):
+    """Gets AI voltage task."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+        yield task
+
+
+@pytest.fixture(scope="module")
+def ao_voltage_task(any_x_series_device):
+    """Gets AO voltage task."""
+    with nidaqmx.Task() as task:
+        task.ao_channels.add_ao_voltage_chan(any_x_series_device.ao_physical_chans[0].name)
+        yield task
+
+
+def test__export_signal__get_enum_property__returns_default_value(ai_voltage_task: Task):
```

Comment:
These tests aren't given an "export signal". Rename to `test__ai_task__...`

Reply:
Renamed all the test cases to `test__ai_task__...`